mirror-index.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -e
+
+INDEX_IMG=""
+INDEX_MIRROR=""
+MIRROR_MANIFEST_FLD="./mirror-index-manifests/"
```

Comment:
```suggestion
MIRROR_MANIFEST_DIR="./mirror-index-manifests/"
```
is more standard terminology I think. In my head, `FLD` = folder.